docs/api/index.rst

```diff
@@ -5,6 +5,6 @@ References
     :maxdepth: 2
 
     models
-    factory
-    pipelines
+    exporter
     runtime
+    graph_ops
```

Comment:
isn't it a little better to put the `graph ops` after `exporter`?

Reply:
ok

Reply:
updated